vembrane/modules/fhir.py

```diff
@@ -174,4 +174,18 @@ def execute(args):
             "assemblies": Assemblies(),
             "chromosomes": Chromosomes(),
         },
+        postprocess=postprocess_fhir_record,
     )
+
+
+def postprocess_fhir_record(record: dict) -> dict:
+    def is_valid(record):
+        if type(record) is dict:
+            return all([is_valid(value) for value in record.values()])
```

Comment:
No need for list comprehensions, generator is fine for `all` (so it can early exit).
```suggestion
            return all(is_valid(value) for value in record.values())
```